app.js

```diff
@@ -106,7 +106,9 @@ async function createEmail(message) {
     return {
       timestamp: message.Timestamp,
       subject: parsed.subject,
-      to: parsed.to.text,
+      to: parsed.to?.text,
```

Comment:
This should null values to not error the server, which is what brought me here in the first place :)

Reply:
How can the email not have a to field?